icpc-wf/ansible/admin.yml

```diff
@@ -35,6 +37,11 @@
       tags: clusterssh
     - role: phpstorm
       tags: phpstorm
+  handlers:
+    - name: clear application application
```

Comment:
Typo: duplicate `application`. Also, I think it's better to put it in a separate handlers file (even if global) and include that, to prevent duplication with `domserver.yml` defining the same handler. See e.g. 4f6eeeb14c819643db356c9cc0978c69c4cb04eb